### PR TITLE
New templates layout

### PR DESCRIPTION
**Pull request by DTadrzak**
Changes in order to introduce new template layout.
Those changes should make a templates more readable and allow to easy
adding new templates to project.
New layout based on two main types (generic and custom). Both types have
independent dir.

### CODE REVIEW

**Comment by DTadrzak** [comment]
@ss7pro 
@PiotrProkop 
@pawelsocha

File: stackanetes/manifest.py [hunk 1]
```diff
@@ -143,10 +144,16 @@ def render(self):
         return data
 
     def _find_template(self):
-        if self.type in GENERIC_TYPES:
-            return "generic-{}.yml.j2".format(self.type)
-        elif self.type == "fluentd-elasticsearch":
-            return "fluentd-elasticsearch.yml.j2"
+        if self.type.count('/') != 1:
```
Comment (pawelsocha) [inline stackanetes/manifest.py]: with more than one / code fail again.

File: stackanetes/manifest.py [hunk 2]
```diff
@@ -143,10 +144,16 @@ def render(self):
...
+            msg = "Invalid format type of {} service. Service type should " \
+                  "have following format (generic|job)/(template_name)."
+            LOG.error(msg)
+            raise KeyError(msg)
+        template_type, template_name = self.type.split('/')
+        if template_type == "generic" and template_name in GENERIC_TYPES:
+            return "generic/{}.yml.j2".format(template_name)
+        elif template_type == "custom" and template_name in CUSTOM_TYPES:
+            return "custom/{}.yml.j2".format(template_name)
         else:
```
Comment (pawelsocha) [inline stackanetes/manifest.py]: For me this else is not necessary (after change elif for if)

File: stackanetes/manifest.py [hunk 3]
```diff
@@ -143,10 +144,16 @@ def render(self):
         return data
 
     def _find_template(self):
-        if self.type in GENERIC_TYPES:
-            return "generic-{}.yml.j2".format(self.type)
-        elif self.type == "fluentd-elasticsearch":
-            return "fluentd-elasticsearch.yml.j2"
+        if self.type.count('/') != 1:
+            msg = "Invalid format type of {} service. Service type should " \
+                  "have following format (generic|job)/(template_name)."
```
Comment (mateusz-blaszkowski) [inline stackanetes/manifest.py]: Did you mean to use `custom` instead of `job` in `(generic|job)/(template_name)` ?

**Pull request merged**